tests/constants.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
```

Comment:
```suggestion
 * Copyright © 2022 Lisk Foundation
```

Reply:
There are many files with 2019. Will refactor all of those at once after applying other comments.

Reply:
As explained by Sameer, updated this one only.